src/Client.php

```diff
@@ -183,7 +192,15 @@ public function getUserAgent(): string
      */
     public function getClientVersion(): string
     {
-        return sprintf('Apigee Edge PHP Client %s', self::VERSION);
+        return sprintf('Apigee Edge PHP Client/%s', self::VERSION);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPHPVersion(): string
```

Comment:
Why this has to be public? Why a new method is needed for this simple system call?